addons/controllers/suite_test.go

```diff
@@ -264,6 +267,28 @@ var _ = BeforeSuite(func(done Done) {
 	)
 	Expect(bootstrapReconciler.SetupWithManager(ctx, mgr, controller.Options{MaxConcurrentReconciles: 1})).To(Succeed())
 
+	// Set up a ClusterCacheTracker to provide to PackageInstallStatus controller which requires a connection to remote clusters
+	l := ctrl.Log.WithName("remote").WithName("ClusterCacheTracker")
+	tracker, err := capiremote.NewClusterCacheTracker(mgr, capiremote.ClusterCacheTrackerOptions{
+		Log: &l,
+		ClientUncachedObjects: []client.Object{
+			&corev1.ConfigMap{},
+			&corev1.Secret{},
+			&corev1.Pod{},
```

Comment:
Should we add other default objects? Is there a way to add an allowed list so that we could just cache PackageInstall and not use a lot of memory

Reply:
@sbueringer how do you think on this? I also wanted to ask if there was a way to only watch resources of specific type or minimize the watched resource types as much as possible?

Reply:
It's currently only possible to define UncachedObjects and not CachedObjects the other way around. This is because `NewClusterCacheTracker` only passes ClientUncachedObjects through to `client.NewDelegatingClient` and that func also only allows to specify UncachedObjects.

But let me give you some context:
* Let's assume no ClientUncachedObjects are set
* When a read call (List/Get) is done on the controller-runtime client (in case of ClusterCacheTracker a client for the workload cluster), the client automatically creates an informer and a cache to cache the objects
* Of course this is "skipped" when the corresponding object is in the ClientUncachedObjects list

This means that only informers/caches are created for objects that you actual Get/List in your code. So while it's not ideal that you have to define the list this way, you only have to care about the objects that you are actually reading.

Reply:
So then considering that I only Get PackageInstall and Package using my cached client, the above list of resources that I've provided for  ```ClientUncachedObjects``` is *more than enough*. I'll not drop the above list of objects in that case to not confuse other code readers.

Reply:
@sbueringer I was wondering considering that I am issuing a ```Get``` call for ```Package``` and ```PackageInstall ``` resources in my workload clusters and I only need the watch for ```PackageInstall``` resource type, does it make sense if I add the ```Package``` type to  ```ClientUncachedObjects```?

```
ClientUncachedObjects: []client.Object{&kappdatapkg.Package{}},
```